docs/specification/1.0.md

```diff
@@ -873,7 +873,9 @@ Hook creators SHOULD include examples of the context.
 If the context contains FHIR data, hook creators SHOULD include examples across multiple versions of FHIR if differences across FHIR versions are possible.
 
 ### Hook Maturity Model
-The intent of the CDS Hooks Maturity Model is to attain broad community engagement and consensus, before a hook is labeled as mature, that the hook is necessary, implementable, and worthwhile to the CDS services and CDS clients that would reasonably be expected to use it. Implementer feedback should drive the maturity of new hooks. Diverse participation in open developer forums and events, such as HL7 FHIR Connectathons, is necessary to achieve significant implementer feedback. The below criteria will be evaluated with these goals in mind.
+The intent of the CDS Hooks Maturity Model is to attain broad community engagement and consensus, before a hook is labeled as mature, that the hook is necessary, implementable, and worthwhile to the CDS services and CDS clients that would reasonably be expected to use it. Implementer feedback should drive the maturity of new hooks. Diverse participation in open developer forums and events, such as HL7 FHIR Connectathons, is necessary to achieve significant implementer feedback. The below criteria will be evaluated with these goals in mind. The hook definition defines the hook's current maturity level and MUST link to the definition of that maturity level.
```

Comment:
> and MUST link to the definition of that maturity level.

I thought the important thing is that a hook definition MUST define its hook maturity level.

I'm not sure we should require the maturity level to be linked. Yes, linking is nice but we don't require it for anything else (like say what "prefetch token" means). I feel like that for the concept of Maturity Model, along with every other concept we have defined in CDS Hooks, requires the reader of the hook to refer back to the spec to understand everything completely.